elementary/monitor/cli.py

```diff
@@ -154,10 +158,12 @@ def get_cli_properties() -> dict:
         return dict()
 
     reload_monitoring_configuration = params.get("reload_monitoring_configuration")
+    target_path = params.get("target_path")
     update_dbt_package = params.get("update_dbt_package")
     full_refresh_dbt_package = params.get("full_refresh_dbt_package")
 
     return {
+        "target_path": target_path,
```

Comment:
Is this necessary?
What does it affect?

Reply:
Are you asking if we need it in the return or if we need the `"target_path": target_path` instead of just returning target_path itself?

For the first question - I'm pretty sure that's how we set the params (instead of it being in the args)